launcher_go/v2/cli_build.go

```diff
@@ -90,6 +90,51 @@ func (r *DockerConfigureCmd) Run(cli *Cli, ctx *context.Context) error {
 	return pups.Run()
 }
 
+type DockerMigrateCmd struct {
+	Config                       string `arg:"" name:"config" help:"config" predictor:"config"`
+	SkipPostDeploymentMigrations bool   `env:"SKIP_POST_DEPLOYMENT_MIGRATIONS" help:"Skip post-deployment migrations. Runs safe migrations only. Defers breaking-change migrations. Make sure you run post-deployment migrations after a full deploy is complete if you use this option."`
+}
+
+func (r *DockerMigrateCmd) Run(cli *Cli, ctx *context.Context) error {
+	config, err := config.LoadConfig(cli.ConfDir, r.Config, true, cli.TemplatesDir)
+	if err != nil {
+		return errors.New("YAML syntax error. Please check your containers/*.yml config files.")
+	}
+	containerId := "discourse-build-" + uuid.NewString()
+	env := []string{"SKIP_EMBER_CLI_COMPILE=1"}
```

Comment:
I'm curious why this is needed when we are only running stuff tagged `db,migrate`.

Reply:
ahh this one's probably not necessary come to think of it, believe you're correct here 👍